modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/odata/entity/v1_0/test/ObjectEntryEntityModelTest.java

```diff
@@ -154,9 +158,14 @@ public void testGetEntityFieldsMap() throws PortalException {
 				"userId",
 				new IntegerEntityField("userId", locale -> Field.USER_ID)
 			).putAll(
-				_getEntityFieldsMap(customObjectFields)
+				_getExpectedEntityFieldsMap(
+					customObjectFields, relatedObjectDefinition,
+					objectRelationship)
 			).build(),
-			objectEntryEntityModel.getEntityFieldsMap());
+			_getObjectDefinitionEntityFieldsMap(objectDefinition));
+
+		_deleteCreatedData(
```

Comment:
NOTE: If you includes this code in here and the test throws a exception, the created data will stays in the database

Reply:
Good catch!!